package/yast2-schema.spec

```diff
@@ -41,7 +41,8 @@ BuildRequires:	trang yast2-devtools yast2-testsuite
 # in /usr/share/YaST2/schema/autoyast/rng/*.rng
 BuildRequires: autoyast2
 BuildRequires: yast2
-BuildRequires: yast2-add-on
+# FIXME: storage-ng
```

Comment:
Yes, in particular, say when it makes sense to enable this again, maybe reference a feature or a bug.